screens/Meal.tsx

```diff
@@ -2,14 +2,209 @@ import {BottomTabScreenProps} from '@react-navigation/bottom-tabs';
 import React, {useState} from 'react';
 import {StyleSheet} from 'react-native';
 import {RootTabList} from '../App';
+import {Box, Center, HStack, ScrollView, Text, VStack} from 'native-base';
+import _ from 'lodash';
 
 type Props = BottomTabScreenProps<RootTabList, 'Meal'>;
 
 export default function Meal({navigation}: Props) {
   // todo
-  return null;
+  //   const favoritedNumbers = DUMMY_MEAL_INFO.map(({isFavorited}) =>
+  //     Number(isFavorited),
+  //   ).reduce((sum, currValue) => sum + currValue);
+  //   const notFavoritedNumbers = DUMMY_MEAL_INFO.length - favoritedNumbers;
+  return (
+    <VStack>
+      <ScrollView>
+        {/* <Text>{DUMMY_MEAL_INFO.length}</Text> */}
+        <Center>
+          {DUMMY_MEAL_INFO.filter(item => item.isFavorited)
+            .sort((a, b) => Number(b.isOperating) - Number(a.isOperating))
```

Comment:
```suggestion
  const [favoriteMeal, notFavoriteMeal] = partition(DUMMY_MEAL_INFO,item=>item.isFavorited);
  const [operatingFavoriteMeal, nonOperatingFavoriteMeal] = partition(favoriteMeal,item=>item.isOperating);
  const [operatingNonFavoriteMeal, nonOperatingNonFavoriteMeal] = partition(favoriteMeal,item=>item.isOperating);
  return (
    <VStack>
      <ScrollView>
        {/* <Text>{DUMMY_MEAL_INFO.length}</Text> */}
        <Center>
          {operatingFavoriteMeal.concat(nonOperatingFavoriteMeal)
```
대충 이런식으로 return 하기 전에 가공하면 될듯?? (Best Practice는 아님)

Reply:
좋은듯 partition을 잘 써보겠음